duolingo.py

```diff
@@ -169,45 +217,54 @@ def get_calendar(self, language_abbr=None):
             return self.user_data.calendar
 
     def get_language_progress(self, lang):
+        """Get informations about user's progression in a language."""
         if not self._is_current_language(lang):
             self._switch_language(lang)
 
-        fields = ['streak', 'language_string', 'level_progress', 'num_skills_learned',
-                  'level_percent', 'level_points', 'points_rank', 'next_level',
-                  'level_left', 'language', 'points', 'fluency_score', 'level']
+        fields = ['streak', 'language_string', 'level_progress',
+                  'num_skills_learned', 'level_percent', 'level_points',
+                  'points_rank', 'next_level', 'level_left', 'language',
+                  'points', 'fluency_score', 'level']
 
         return self._make_dict(fields, self.user_data.language_data[lang])
 
     def get_friends(self):
+        """Get user's friends."""
         for k, v in self.user_data.language_data.iteritems():
             data = []
             for friend in v['points_ranking_data']:
-                temp = {'username': friend['username'], 'points': friend['points_data']['total']}
-                temp['languages'] = [i['language_string'] for i in friend['points_data']['languages']]
+                temp = {'username': friend['username'],
+                        'points': friend['points_data']['total']}
+                temp['languages'] = [i['language_string'] for i in
```

Comment:
I think we might be able to just merge this with the rest of the `temp` dict instead of adding a new key this way